### PR TITLE
Create minio bucket using init container

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Workaround for https://github.com/minio/operator/issues/1346

### CODE REVIEW

File: developer/config.yaml [hunk 1]
```diff
@@ -6,10 +6,10 @@
 cluster_type: openshift
 
 # git_url refers to a git repo to be considered as the source of truth for Argo CD applications.
-git_url: https://github.com/openshift-pipelines/pipeline-service.git
+git_url: https://github.com/AndrienkoAleksandr/pipeline-service.git
```
Comment (bnallapeta) [inline developer/config.yaml:9]: Need to revert this here and at multiple other places.

File: developer/openshift/gitops/argocd/pipeline-service/tekton-results/minio-bucket-creation.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (bnallapeta) [inline developer/openshift/gitops/argocd/pipeline-service/tekton-results/minio-bucket-creation.yaml:1]: This file shares a lot in common with [s3-configuration-patch.yaml](https://github.com/openshift-pipelines/pipeline-service/blob/main/developer/openshift/gitops/argocd/pipeline-service/tekton-results/s3-configuration-patch.yaml). Should it not be the case that only one of the two files should exist?

**Review by bnallapeta — COMMENTED** [review]
- Have provided few comments on the PR.
- Need to squash the commits.
- Static analysis has failed.

File: developer/openshift/gitops/argocd/pipeline-service/tekton-results/minio-bucket-creation.yaml (hunk 2 above)
Comment (AndrienkoAleksandr) [reply developer/openshift/gitops/argocd/pipeline-service/tekton-results/minio-bucket-creation.yaml]: ok, I moved this part to the same file.

**Review by Roming22 — CHANGES_REQUESTED** [review]
Can you please rebase to main and squash the history?

**Review by Roming22 — APPROVED** [review]

**Review by bnallapeta — APPROVED** [review]

**Pull request merged**